From dd6ba769ea8b411d6e09720dfe7f38d027d8f8fc Mon Sep 17 00:00:00 2001
From: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Date: Tue, 28 Apr 2015 09:31:35 +0200
Subject: [PATCH] Fix packageconfig path. Rationale: LIB_DESTINATION is set as
 CMAKE_INSTALL_FULL_LIBDIR. So repeating "{prefix}" results in a double
 usr/usr inclusion

---
 liblucene++-contrib.pc.cmake | 4 ++--
 liblucene++.pc.cmake         | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/liblucene++-contrib.pc.cmake b/liblucene++-contrib.pc.cmake
index 98b6381..21026e0 100644
--- a/liblucene++-contrib.pc.cmake
+++ b/liblucene++-contrib.pc.cmake
@@ -1,13 +1,13 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}/bin
-libdir=${prefix}/@LIB_DESTINATION@
+libdir=@LIB_DESTINATION@
 includedir=${prefix}/include/lucene++
 lib=lucene++-contrib
 
 Name: liblucene++-contrib
 Description: Contributions for Lucene++ - a C++ search engine, ported from the popular Apache Lucene
 Version: @lucene++_VERSION@
-Libs: -L${prefix}/@LIB_DESTINATION@ -l${lib}
+Libs: -L@LIB_DESTINATION@ -l${lib}
 Cflags: -I${includedir}
 Requires: liblucene++ = @lucene++_VERSION@
 
diff --git a/liblucene++.pc.cmake b/liblucene++.pc.cmake
index c526d4a..32d16ad 100644
--- a/liblucene++.pc.cmake
+++ b/liblucene++.pc.cmake
@@ -1,12 +1,12 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}/bin
-libdir=${prefix}/@LIB_DESTINATION@
+libdir=@LIB_DESTINATION@
 includedir=${prefix}/include/lucene++
 lib=lucene++
 
 Name: liblucene++
 Description: Lucene++ - a C++ search engine, ported from the popular Apache Lucene
 Version: @lucene++_VERSION@
-Libs: -L${prefix}/@LIB_DESTINATION@ -l${lib}
+Libs: -L@LIB_DESTINATION@ -l${lib}
 Cflags: -I${includedir}
 
